packages/core/src/types.ts

```diff
@@ -48,6 +48,7 @@ export class Page {
   public settings: Settings = {};
   public id: string = "";
   public path: string = "";
+  public title: string = "";
```

Comment:
Please remove the title from the page. This should be part of the head attribute.